instrumentation/otel-webserver-module/common.gradle

```diff
@@ -8,6 +8,9 @@ defaultTasks 'build'
 ext {
   ARCH = ''
```

Comment:
Is this used anywhere? I think I've added a case statement to have it work for ARM64